frame/conviction-voting/src/lib.rs

```diff
@@ -58,44 +62,43 @@ pub mod benchmarking;
 
 const CONVICTION_VOTING_ID: LockIdentifier = *b"pyconvot";
 
-type BalanceOf<T> =
-	<<T as Config>::Currency as Currency<<T as frame_system::Config>::AccountId>>::Balance;
-type VotingOf<T> = Voting<
-	BalanceOf<T>,
+type BalanceOf<T, I = ()> =
+	<<T as Config<I>>::Currency as Currency<<T as frame_system::Config>::AccountId>>::Balance;
+type VotingOf<T, I> = Voting<
```

Comment:
I am with @KiChjang here. to rephrase, why not use `I = ()` _everywhere_?  In that case you would have no breaking change either.

Reply:
I don't wanna argue this pointless question anymore, so I added default type param